src/theme/colors.js

```diff
@@ -27,7 +26,7 @@ const colors = {
   summerSky: "#3AC7FF",
   blackPearl: "#1F262D",
   slate: "#999999",
-  moonstone: "#D1D1D1",
+  f: "#D1D1D1",
```

Comment:
I think you renamed this color by mistake

Reply:
fixed